test/MUXControlsTestApp/App.xaml

```diff
@@ -18,7 +18,8 @@
                 </ResourceDictionary>
             </ResourceDictionary.ThemeDictionaries>
             <ResourceDictionary.MergedDictionaries>
-                <controls:XamlControlsResources/>
+                <controls:XamlControlsResources UseCompactDictionary="True"/>
```

Comment:
I assume you had this on for testing. I wouldn't think we'd want it on by default.

Reply:
For test purpose, and now it's removed